### PR TITLE
[MO] Deploy project to production - android

### PR DESCRIPTION
**Pull request by yassinecc**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/mo-deploy-to-production-android/react-native/setup/deploy-to-production-android.mo.md">deploy-to-production-android.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by tychota — CHANGES_REQUESTED** [review]
Hello @yassinecc  , I fixed a bug I introduced working on dangerfiles: can you rebase https://github.com/bamlab/dev-standards/commit/6ebbc236cd6d51410519f78c03c62a10e48effc3

I can then help you reviewing.

File: react-native/setup/deploy-to-production-android.mo.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Your client admin account on Google Play Console
+
+## Steps
+
+### 1. Bump version
+Bump version code and version name in `fastlane/.env.prod`.
+
+### 2. Launch the build process
+This will build your app and upload it to the Google Play Console.
+```bash
```
Comment (tychota) [inline react-native/setup/deploy-to-production-android.mo.md]: Extra ligne before code, else gitbook will behave weirdly.

File: react-native/setup/deploy-to-production-android.mo.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+bundle exec fastlane android deploy --env=production
+```
+
+### 6. Make your app available for beta testing
```
Comment (tychota) [inline react-native/setup/deploy-to-production-android.mo.md]: Where are part 3 -> 6

**Review by tychota — CHANGES_REQUESTED** [review]

**Comment by xavierlefevre** [comment]
@yassinecc le rebase c'est bien passé ? car je ne vois plus aucun fichier modifié dans la PR :O

**Comment by yassinecc** [comment]
@tychota Can you review this PR?

File: react-native/setup/deploy-to-production-android.mo.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+```bash
+bundle exec fastlane android deploy --env=production
+```
+
+### 3. Make your app available for beta testing
+
+* Log in to [the developer console](https://play.google.com/apps/publish/) with your project's account and select your app
+* Go to `Release Management/App releases` in the left panel
+* Click on `Manage Beta` then `Create release` and upload the APK generated from the build process. This file is located at `android/app/build/outputs/apk/app-release.apk`
```
Comment (Minishlink) [inline react-native/setup/deploy-to-production-android.mo.md:28]: This can be automated using fastlane ;)

**Review by tychota — APPROVED** [review]

**Pull request merged**